docs/strategies/circuit-breaker.md

```diff
@@ -37,6 +37,19 @@ new ResiliencePipelineBuilder().AddCircuitBreaker(new CircuitBreakerStrategyOpti
     ShouldHandle = new PredicateBuilder().Handle<SomeExceptionType>()
 });
 
+// Adds a circuit breaker with dynamic break duration:
```

Comment:
Please do not edit code snippets in the markdown files directly.
- They are copied over from a compile-able cs file.
- In case of `circuit-breaker.md` the related code file is the `/src/Snippets/Docs/CircuitBreaker.cs`

For more information please checkout the [Snippets readme](https://github.com/App-vNext/Polly/blob/main/src/Snippets/README.md).

Reply:
I've taken note of your guidance not to edit code snippets in the markdown files directly. I've made the necessary additions to the Usage in the /src/Snippets/Docs/CircuitBreaker.cs file as instructed.